CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+## Reporting Issues
+
+If you have found what you think is a bug, please [file an issue](https://github.com/pmndrs/zustand/issues/new).
```

Comment:
Lately, we ask to open a discussion first before filing an issue. We should also fix the contribution guide in [jotai](https://github.com/pmndrs/jotai).